user-service/user-domain/src/main/kotlin/com/grepp/quizy/domain/user/UserAppender.kt

```diff
@@ -0,0 +1,12 @@
+package com.grepp.quizy.domain.user
+
+import org.springframework.stereotype.Component
+
+@Component
+class UserAppender (
+        private val userRepository: UserRepository
+) {
+    fun appendUser(user: User): User {
+        return userRepository.save(user)
+    }
```

Comment:
```suggestion
    fun appendUser(user: User): User = userRepository.save(user)
```

이거 민혁님 PR에서도 코멘트 남긴거긴 한데
Kotlin 코딩 컨벤션에서는 단일 return문만 있는 함수는 그냥 저렇게 작성하는 걸 지향한다고 합니다!

Reply:
넵 코틀린이 익숙하지 않아서 편한대로 작성했네요
지적 감사합니다..!